handler/src/main/java/io/netty/handler/ssl/util/SelfSignedCertificate.java

```diff
@@ -72,6 +75,15 @@
      * Creates a new instance.
      */
     public SelfSignedCertificate() throws CertificateException {
+        this(DEFAULT_NOT_BEFORE, DEFAULT_NOT_AFTER);
+    }
+
+    /**
+     * Creates a new instance.
+     * @param notBefore Certificate is not valid before this time
+     * @param notAfter Certificate is not valid after this time
+     */
+    public SelfSignedCertificate(Date notBefore, Date notAfter) throws CertificateException {
         this("example.com");
```

Comment:
Pushed a new set of commits. See https://github.com/netty/netty/pull/4257#issuecomment-142764968